src/pylibsshext/sftp.pyx

```diff
@@ -54,3 +54,30 @@ cdef class SFTP:
                     raise LibsshSFTPException(self, "Writing to remote file [%s] failed" % remote_file)
                 buffer = f.read(1024)
             sftp.sftp_close(rf)
+
+    def get(self, remote_file, local_file):
+        cdef sftp.sftp_file rf
+        if remote_file in ['', None]:
```

Comment:
This check is not required. As if the file in not found sftp exception will be raised

Reply:
It worked. :+1: